modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -62,6 +64,9 @@ object GlobalSnapshotConsensusFunctions {
           metrics.globalSnapshot(signedArtifact),
           logger.error("Cannot save GlobalSnapshot into the storage")
         )
+        .flatTap { _ =>
+          gossipForkInfo(gossip, signedArtifact)
+        }
```

Comment:
@sdavidp  why such change? IMO changing the order of execution may drastically obfuscate the code. Also, there is a difference between *> and >> and we prefer the later as it chain operations

Reply:
@marcinwadon You're right. I thought it was equivalent to _flatTap_ but it's not, it's still calling _productL_ which is not the same. `<<` is deprecated which is why I used `<*`.

I think the deprecation message is misleading because the operations are not analagous:
`>>` is an alias for `flatMap` but `<<` is NOT an alias for `flatTap` as I expected.

I'm sorry @cngo-github, @marcinwadon  is correct. Ignore those two suggestions to use `<*`

Reply:
Changed the `flatTap`s to `>>`.